docs/extensions/heartbeat.md

```diff
@@ -0,0 +1,6 @@
+# Heartbeat Controller
+
+The heartbeat controller renews a dedicated `Lease` object named `gardener-extension-heartbeat` at regular 30 second intervals by default. This `Lease` is used for heart beats similar to how `gardenlet` uses `Lease` objects for seed heart beats (see [gardenlet heartbeats](../concepts/gardenlet#heartbeats)).
```

Comment:
Sometimes you write "heart beat", sometimes "heartbeat". I think the latter is correct. Can you please adapt this everywhere (also in code, not only here)?